# Italian translation for update-manager
# Copyright (c) (c) 2005 Canonical Ltd, and Rosetta Contributors 2005
# This file is distributed under the same license as the update-manager package.
# Fabio Marzocca <thesaltydog@gmail.com>, 2005.
#
#
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2012-06-14 00:53+0100\n"
"PO-Revision-Date: 2012-03-21 17:44+0000\n"
"Last-Translator: Alessandro Ranaldi <ranaldialessandro@gmail.com>\n"
"Language-Team: Italian <it@li.org>\n"
"Language: it\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2012-04-17 13:41+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:433 ../UpdateManager/Core/utils.py:433
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f kB"
msgstr[1] "%(size).0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:436 ../UpdateManager/Core/utils.py:436
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:436
#, python-format
msgid "Server for %s"
msgstr "Server in %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Server principale"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Server personalizzati"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Impossibile calcolare la voce di sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Impossibile trovare alcun pacchetto. È probabile che non sia stato "
"selezionato un disco di Ubuntu o che l'architettura installata sia "
"differente."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Aggiunta del CD non riuscita"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Si è verificato un errore nell'aggiungere il CD, l'avanzamento di versione "
"verrà interrotto. Segnalare questo evento come un bug se si tratta di un CD "
"Ubuntu valido.\n"
"\n"
"Il messaggio di errore è stato:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:151
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Rimuovi il pacchetto danneggiato"
msgstr[1] "Rimuovi i pacchetti danneggiati"

#: ../DistUpgrade/DistUpgradeCache.py:154
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Il pacchetto «%s» è in uno stato inconsistente e deve essere reinstallato, "
"ma non è stato trovato alcun archivio. Rimuovere il pacchetto per continuare?"
msgstr[1] ""
"I pacchetti «%s» sono in uno stato inconsistente e devono essere "
"reinstallati, ma non è stato trovato alcun archivio. Rimuovere i pacchetti "
"per continuare?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:255
msgid "The server may be overloaded"
msgstr "Il server potrebbe essere sovraccarico"

#: ../DistUpgrade/DistUpgradeCache.py:368
msgid "Broken packages"
msgstr "Pacchetti danneggiati"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Il sistema contiene pacchetti non integri che non possono essere corretti "
"con questo software. Prima di procedere, correggere tali pacchetti con "
"«synaptic» o «apt-get»."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Si è verificato un problema durante il calcolo dell'avanzamento:\n"
"%s\n"
"\n"
" Le cause possono essere:\n"
" * Avanzamento a una versione di pre-rilascio di Ubuntu\n"
" * Utilizzo di una versione di pre-rilascio di Ubuntu\n"
" * Pacchetti software non ufficiali non forniti da Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:703
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Si tratta molto probabilmente di un problema momentaneo, riprovare in un "
"secondo momento."

#: ../DistUpgrade/DistUpgradeCache.py:706
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Se nessuna delle precedenti è la causa, segnalare questo problema tramite il "
"comando «ubuntu-bug update-manager» da terminale."

#: ../DistUpgrade/DistUpgradeCache.py:711
#: ../UpdateManager/UpdateManager.py:1031
msgid "Could not calculate the upgrade"
msgstr "Impossibile calcolare l'avanzamento"

#: ../DistUpgrade/DistUpgradeCache.py:762
msgid "Error authenticating some packages"
msgstr "Errore nell'autenticare alcuni pacchetti"

#: ../DistUpgrade/DistUpgradeCache.py:763
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Non è stato possibile autenticare alcuni pacchetti. Questo potrebbe essere "
"un problema di rete passeggero, è possibile riprovare più tardi. Segue "
"l'elenco dei pacchetti non autenticati."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Il pacchetto «%s» è selezionato per la rimozione, ma è nella blacklist per "
"la rimozione."

#: ../DistUpgrade/DistUpgradeCache.py:787
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Il pacchetto essenziale «%s» è selezionato per la rimozione."

#: ../DistUpgrade/DistUpgradeCache.py:796
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Tentativo di installazione della versione «%s» presente nella blacklist"

#: ../DistUpgrade/DistUpgradeCache.py:914
#, python-format
msgid "Can't install '%s'"
msgstr "Impossibile installare «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Non è stato possibile installare un pacchetto richiesto. Segnalare questo "
"problema tramite il comando «ubuntu-bug update-manager» da terminale."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:926
msgid "Can't guess meta-package"
msgstr "Impossibile indovinare il meta-pacchetto"

#: ../DistUpgrade/DistUpgradeCache.py:927
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Il sistema non contiene alcun pacchetto «ubuntu-desktop», «kubuntu-desktop», "
"«xubuntu-desktop» oppure «edubuntu-desktop» e non è stato possibile rilevare "
"la versione di Ubuntu in esecuzione.\n"
" Prima di procedere, usare «synaptic» o «apt-get» per installare uno dei "
"pacchetti sopra menzionati."

#: ../DistUpgrade/DistUpgradeController.py:114
msgid "Reading cache"
msgstr "Lettura della cache"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid "Unable to get exclusive lock"
msgstr "Impossibile ottenere un blocco esclusivo"

#: ../DistUpgrade/DistUpgradeController.py:224
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Questo solitamente significa che un'altra applicazione di gestione dei "
"pacchetti (come «apt-get» o «aptitude») è già in esecuzione. Chiudere "
"l'altra applicazione prima di continuare."

#: ../DistUpgrade/DistUpgradeController.py:257
msgid "Upgrading over remote connection not supported"
msgstr "Avanzamento tramite connessione remota non supportato"

#: ../DistUpgrade/DistUpgradeController.py:258
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Si sta effettuando l'avanzamento tramite una connessione SSH con "
"un'interfaccia che non la supporta. Provare a eseguire un avanzamento in "
"modalità testuale con il comando «do-release-upgrade».\n"
"\n"
"L'avanzamento è stato interrotto. Riprovare senza SSH."

#: ../DistUpgrade/DistUpgradeController.py:272
msgid "Continue running under SSH?"
msgstr "Continuare la sessione con SSH?"

#: ../DistUpgrade/DistUpgradeController.py:273
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Questa sessione sembra essere stata avviata con SSH. Non è consigliato "
"eseguire un avanzamento con SSH perché, in caso di errore, il recupero "
"risulta più difficile.\n"
"\n"
"Continuando verrà avviato un demone SSH aggiuntivo sulla porta «%s».\n"
"Continuare?"

#: ../DistUpgrade/DistUpgradeController.py:287
msgid "Starting additional sshd"
msgstr "Avvio demone sshd addizionale"

#: ../DistUpgrade/DistUpgradeController.py:288
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Per facilitare il ripristino in caso di errori, verrà avviato un demone sshd "
"aggiuntivo sulla porta «%s». Qualora si verifichino malfunzionamenti con il "
"servizio ssh in esecuzione, sarà ancora possibile collegarsi a quello "
"aggiuntivo.\n"

#: ../DistUpgrade/DistUpgradeController.py:296
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Se si sta utilizzando un firewall, potrebbe essere necessario aprire questa "
"porta temporaneamente. Questa operazione non viene effettuata "
"automaticamente perché è potenzialmente pericolosa. Un possibile comando per "
"aprire la porta è:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:368
#: ../DistUpgrade/DistUpgradeController.py:413
msgid "Can not upgrade"
msgstr "Impossibile eseguire l'avanzamento"

#: ../DistUpgrade/DistUpgradeController.py:369
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"Questo strumento non supporta l'avanzamento dalla versione «%s» alla "
"versione «%s»."

#: ../DistUpgrade/DistUpgradeController.py:378
msgid "Sandbox setup failed"
msgstr "Impostazione della sandbox non riuscita."

#: ../DistUpgrade/DistUpgradeController.py:379
msgid "It was not possible to create the sandbox environment."
msgstr "Impossibile creare l'ambiente sanbox."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid "Sandbox mode"
msgstr "Modalità sandbox"

#: ../DistUpgrade/DistUpgradeController.py:386
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Questo avanzamento viene eseguito in modalità \"sandbox\" (di prova). Tutte "
"le modifiche vengono scritte in «%s» e al prossimo riavvio andranno perse.\n"
"\n"
"Nessuna modifica salvata in una cartella di sistema, da ora al prossimo "
"riavvio, è permanente."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"La propria installazione di python è danneggiata. Correggere il collegamento "
"simbolico \"/usr/bin/python\"."

#: ../DistUpgrade/DistUpgradeController.py:440
msgid "Package 'debsig-verify' is installed"
msgstr "Il pacchetto «debsig-verify» è installato"

#: ../DistUpgrade/DistUpgradeController.py:441
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Non è possibile eseguire l'avanzamento con tale pacchetto installato.\n"
"Rimuoverlo usando Synaptic oppure col comando \"apt-get remove debsig-verify"
"\" quindi eseguire nuovamente l'avanzamento."

#: ../DistUpgrade/DistUpgradeController.py:453
#, python-format
msgid "Can not write to '%s'"
msgstr "Impossibile scrivere su «%s»"

#: ../DistUpgrade/DistUpgradeController.py:454
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Impossibile scrivere nella directory di sistema «%s». L'aggiornamento non "
"può continuare.\n"
"Assicurarsi che la directory di sistema sia accessibile in scrittura."

#: ../DistUpgrade/DistUpgradeController.py:465
msgid "Include latest updates from the Internet?"
msgstr "Includere ultimi aggiornamenti da Internet?"

#: ../DistUpgrade/DistUpgradeController.py:466
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Gli aggiornamenti più recenti possono essere scaricati automaticamente da "
"Internet e installati durante l'avanzamento. Se è presente una connessione "
"di rete, questo è altamente consigliato.\n"
"\n"
"L'avanzamento durerà più a lungo, ma quando sarà completo, il sistema sarà "
"completamente aggiornato. È possibile non farlo, ma è comunque consigliato "
"installare gli ultimi aggiornamenti al termine dell'avanzamento.\n"
"Rispondendo «no», la rete non verrà utilizzata."

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid "disabled on upgrade to %s"
msgstr "disabilitato durante l'avanzamento a %s"

#: ../DistUpgrade/DistUpgradeController.py:713
msgid "No valid mirror found"
msgstr "Non è stato trovato alcun mirror valido"

#: ../DistUpgrade/DistUpgradeController.py:714
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Durante l'analisi delle informazioni sui repository non è stata trovata "
"alcuna voce di mirror per l'avanzamento. Questo può essere provocato "
"dall'esecuzione del mirror in locale o da dati sui mirror non recenti.\n"
"\n"
"Sovrascrivere il file «sources.list» comunque? Scegliendo «Sì» verranno "
"sostituite le voci «%s» in «%s».\n"
"Scegliendo «No» l'avanzamento verrà annullato."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:734
msgid "Generate default sources?"
msgstr "Generare le sorgenti predefinite?"

#: ../DistUpgrade/DistUpgradeController.py:735
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Non è stata trovata alcuna voce per «%s» nel file «sources.list».\n"
"\n"
"Aggiungere le voci predefinite per «%s»? Scegliendo «No» l'avanzamento verrà "
"annullato."

#: ../DistUpgrade/DistUpgradeController.py:770
msgid "Repository information invalid"
msgstr "Informazioni sul repository non valide"

#: ../DistUpgrade/DistUpgradeController.py:771
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"L'aggiornamento delle informazioni sui repository ha creato un file non "
"valido. Sarà ora possibile segnalare questo problema."

#: ../DistUpgrade/DistUpgradeController.py:778
msgid "Third party sources disabled"
msgstr "Sorgenti di terze parti disabilitate"

#: ../DistUpgrade/DistUpgradeController.py:779
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Sono state disabilitate alcune voci di terze parti nel file «sources.list». "
"È possibile abilitarle di nuovo dopo l'avanzamento di versione con lo "
"strumento «software-properties» o con il gestore di pacchetti."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pacchetto in uno stato inconsistente"
msgstr[1] "Pacchetti in uno stato inconsistente"

#: ../DistUpgrade/DistUpgradeController.py:822
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Il pacchetto «%s» è in uno stato inconsistente e deve essere reinstallato, "
"ma non è stato trovato alcun archivio. Reinstallare il pacchetto manualmente "
"o rimuoverlo dal sistema."
msgstr[1] ""
"I pacchetti «%s» sono in uno stato inconsistente e devono essere "
"reinstallati, ma non è stato trovato alcun archivio. Reinstallare i "
"pacchetti manualmente o rimuoverli dal sistema."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Error during update"
msgstr "Errore durante l'aggiornamento"

#: ../DistUpgrade/DistUpgradeController.py:871
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Si è verificato un problema durante l'aggiornamento. Solitamente si tratta "
"di problemi di rete, controllare la connessione di rete e riprovare."

#. print("on_button_install_clicked")
#: ../DistUpgrade/DistUpgradeController.py:880
#: ../UpdateManager/UpdateManager.py:757
msgid "Not enough free disk space"
msgstr "Spazio libero su disco insufficiente"

#: ../DistUpgrade/DistUpgradeController.py:881
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"L'avanzamento si è interrotto: sono necessari %s di spazio libero sul disco "
"«%s», liberare almeno altri %s di spazio sul disco «%s». Svuotare il cestino "
"e rimuovere pacchetti temporanei di installazioni precedenti con il comando "
"«sudo apt-get clean»."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:910
#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Calculating the changes"
msgstr "Calcolo delle modifiche"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:942
msgid "Do you want to start the upgrade?"
msgstr "Avviare l'avanzamento di versione?"

#: ../DistUpgrade/DistUpgradeController.py:1008
msgid "Upgrade canceled"
msgstr "Avanzamento annullato"

#: ../DistUpgrade/DistUpgradeController.py:1009
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"L'avanzamento di versione verrà annullato e sarà ripristinato lo stato "
"originale del sistema. È possibile riprendere l'avanzamento di versione in "
"un secondo momento."

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1149
msgid "Could not download the upgrades"
msgstr "Impossibile scaricare gli avanzamenti di versione"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"L'avanzamento si è interrotto. Controllare la connessione a Internet o il "
"supporto di installazione e riprovare. Tutti i file scaricati finora saranno "
"conservati."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1100
#: ../DistUpgrade/DistUpgradeController.py:1137
#: ../DistUpgrade/DistUpgradeController.py:1242
msgid "Error during commit"
msgstr "Errore durante il commit"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1102
#: ../DistUpgrade/DistUpgradeController.py:1139
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid "Restoring original system state"
msgstr "Ripristino dello stato originale del sistema"

#: ../DistUpgrade/DistUpgradeController.py:1103
#: ../DistUpgrade/DistUpgradeController.py:1118
#: ../DistUpgrade/DistUpgradeController.py:1140
msgid "Could not install the upgrades"
msgstr "Impossibile installare gli avanzamenti di versione"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"L'avanzamento si è interrotto: il sistema potrebbe essere in uno stato "
"inutilizzabile. Verrà avviato un ripristino (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1113
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/update-manager/+filebug and attach the files in /var/log/dist-"
"upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Segnalare questo problema in un browser all'indirizzo http://bugs.launchpad."
"net/ubuntu/+source/update-manager/+filebug e includere i file contenuti in /"
"var/log/dist-upgrade/ nella segnalazione.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1150
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"L'avanzamento si è interrotto: controllare la connessione a Internet o il "
"supporto di installazione e riprovare. "

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid "Remove obsolete packages?"
msgstr "Rimuovere i pacchetti obsoleti?"

#: ../DistUpgrade/DistUpgradeController.py:1231
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Mantieni"

#: ../DistUpgrade/DistUpgradeController.py:1231
msgid "_Remove"
msgstr "_Rimuovi"

#: ../DistUpgrade/DistUpgradeController.py:1243
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Si è verificato un problema durante la pulizia. Leggere il messaggio "
"seguente per maggiori informazioni. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1319
msgid "Required depends is not installed"
msgstr "Le dipendenze richieste non sono installate"

#: ../DistUpgrade/DistUpgradeController.py:1320
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dipendenza richiesta «%s» non è installata. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1588
#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Checking package manager"
msgstr "Controllo gestore dei pacchetti"

#: ../DistUpgrade/DistUpgradeController.py:1593
msgid "Preparing the upgrade failed"
msgstr "Preparazione dell'avanzamento di versione non riuscita"

#: ../DistUpgrade/DistUpgradeController.py:1594
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"La preparazione del sistema per l'avanzamento non è riuscita. Sarà ora "
"possibile segnalare questo problema."

#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Getting upgrade prerequisites failed"
msgstr "Recupero dei prerequisiti per l'avanzamento di versione non riuscito"

#: ../DistUpgrade/DistUpgradeController.py:1609
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Non è stato possibile ottenere i prerequisiti per l'avanzamento. Verrà ora "
"ripristinato lo stato originario del sistema.\n"
"\n"
"Sarà ora possibile segnalare questo problema."

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "Updating repository information"
msgstr "Aggiornamento delle informazioni sui repository"

#: ../DistUpgrade/DistUpgradeController.py:1644
msgid "Failed to add the cdrom"
msgstr "Aggiunta CD-ROM non riuscita"

#: ../DistUpgrade/DistUpgradeController.py:1645
msgid "Sorry, adding the cdrom was not successful."
msgstr "L'aggiunta del CD-ROM non è avvenuta con successo."

#: ../DistUpgrade/DistUpgradeController.py:1673
msgid "Invalid package information"
msgstr "Informazioni di pacchetto non valide"

#: ../DistUpgrade/DistUpgradeController.py:1674
msgid "After updating your package "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1698
#: ../DistUpgrade/DistUpgradeController.py:1750
msgid "Fetching"
msgstr "Recupero file"

#: ../DistUpgrade/DistUpgradeController.py:1704
#: ../DistUpgrade/DistUpgradeController.py:1754
msgid "Upgrading"
msgstr "Avanzamento versione"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1709
#: ../DistUpgrade/DistUpgradeController.py:1756
#: ../DistUpgrade/DistUpgradeController.py:1763
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Upgrade complete"
msgstr "Avanzamento di versione completato"

#: ../DistUpgrade/DistUpgradeController.py:1710
#: ../DistUpgrade/DistUpgradeController.py:1757
#: ../DistUpgrade/DistUpgradeController.py:1764
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"L'avanzamento di versione è stato completato, ma durante l'operazione si "
"sono verificati degli errori."

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Searching for obsolete software"
msgstr "Ricerca di software obsoleto"

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "System upgrade is complete."
msgstr "L'avanzamento di versione del sistema è stato completato."

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid "The partial upgrade was completed."
msgstr "L'avanzamento parziale è stato completato."

#: ../DistUpgrade/DistUpgradeQuirks.py:204
msgid "evms in use"
msgstr "È in uso evms"

#: ../DistUpgrade/DistUpgradeQuirks.py:205
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Il sistema sta utilizzando il gestore di volumi «evms» in /proc/mounts. Il "
"software «evms» non è più supportato, disattivarlo e rieseguire "
"l'avanzamento."

#: ../DistUpgrade/DistUpgradeQuirks.py:502
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"La scheda video potrebbe non essere pienamente supportata in Ubuntu 12.04 "
"LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:504
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Il supporto per le schede video Intel in Ubuntu 11.10 è limitato e "
"potrebbero esserci dei problemi dopo l'avanzamento. Per ulteriori "
"informazioni andare su https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx Continuare?"

#: ../DistUpgrade/DistUpgradeQuirks.py:526
#: ../DistUpgrade/DistUpgradeQuirks.py:554
#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"L'avanzamento di versione potrebbe compromettere gli effetti visivi e le "
"prestazioni di giochi e di altri programmi che fanno un uso intensivo della "
"grafica."

#: ../DistUpgrade/DistUpgradeQuirks.py:530
#: ../DistUpgrade/DistUpgradeQuirks.py:558
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Sono in uso i driver grafici NVidia «nvidia». In Ubuntu 10.04 LTS non ci "
"sono driver funzionanti con la scheda video in uso.\n"
"\n"
"Continuare comunque?"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Sono in uso i driver grafici AMD «fglrx». In Ubuntu 10.04 LTS non ci sono "
"driver funzionanti con la scheda video in uso.\n"
"\n"
"Continuare comunque?"

#: ../DistUpgrade/DistUpgradeQuirks.py:615
msgid "No i686 CPU"
msgstr "CPU non compatibile"

#: ../DistUpgrade/DistUpgradeQuirks.py:616
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Questo sistema è provvisto di un processore (o CPU) di tipo i586 oppure che "
"non presenta l'estensione \"cmov\". Tutti i pacchetti sono stati creati con "
"ottimizzazioni che richiedono come architettura minima quella di tipo i686. "
"A causa di tale configurazione hardware non è possibile aggiornare questo "
"sistema a un nuovo rilascio di Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:652
msgid "No ARMv6 CPU"
msgstr "Nessuna CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:653
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Il sistema utilizza una CPU ARM antecedente all'architettura ARMv6. Tutti i "
"pacchetti nella versione 9.10 sono stati creati con ottimizzazioni che "
"richiedono ARMv6 come architettura minima. Non è possibile avanzare il "
"sistema a un nuovo rilascio di Ubuntu con l'hardware attuale."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "No init available"
msgstr "\"init\" non disponibile"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Il proprio sistema sembra essere un ambiente di virtualizzazione privo di un "
"demone \"init\" (come Linux-VServer). Ubuntu 10.04 LTS non può funzionare in "
"questo tipo di ambiente ed è prima richiesto un aggiornamento della macchina "
"virtuale.\n"
"\n"
"Continuare comunque?"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Sandbox upgrade using aufs"
msgstr "Avanzamento in ambiente sandbox con file system aufs"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Utilizzare il percorso fornito per cercare un CD-ROM contenente pacchetti da "
"far avanzare."

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Frontend da usare. Disponibili al momento: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "*DEPRECATED* this option will be ignored"
msgstr "*DEPRECATA* Questa opzione sarà ignorata"

#: ../DistUpgrade/DistUpgradeMain.py:79
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Esegue solo un avanzamento parziale (nessuna riscrittura di sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Disable GNU screen support"
msgstr "Disabilita la schermata GNU di supporto"

#: ../DistUpgrade/DistUpgradeMain.py:84
msgid "Set datadir"
msgstr "Imposta datadir"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:195
#: ../UpdateManager/DistUpgradeFetcherKDE.py:155
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Inserire «%s» nell'unità «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:209
msgid "Fetching is complete"
msgstr "Recupero file completato"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:222
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Recupero file %li di %li a %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:296
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:309
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:371
#, python-format
msgid "About %s remaining"
msgstr "Circa %s rimanenti"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:225
#, python-format
msgid "Fetching file %li of %li"
msgstr "Recupero file %li di %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
msgid "Applying changes"
msgstr "Applicazione delle modifiche"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:212
#: ../DistUpgrade/DistUpgradeViewKDE.py:275
msgid "dependency problems - leaving unconfigured"
msgstr "problemi con le dipendenze - lasciato non configurato"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:277
#, python-format
msgid "Could not install '%s'"
msgstr "Impossibile installare «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:278
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"L'avanzamento procederà, ma il pacchetto «%s» potrebbe non essere in uno "
"stato funzionante. Segnalare questo evento come bug."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Sostituire il file di configurazione personalizzato\n"
"«%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Se si decide di sostituire il file di configurazione con una versione più "
"recente, tutte le modifiche apportate andranno perse."

#: ../DistUpgrade/DistUpgradeViewGtk.py:251
#: ../DistUpgrade/DistUpgradeViewGtk3.py:256
#: ../DistUpgrade/DistUpgradeViewKDE.py:323
msgid "The 'diff' command was not found"
msgstr "Il comando «diff» non è stato trovato"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:477
#: ../DistUpgrade/DistUpgradeViewText.py:92
msgid "A fatal error occurred"
msgstr "Si è verificato un errore fatale"

#: ../DistUpgrade/DistUpgradeViewGtk.py:465
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Se non è già stato fatto, segnalare questo bug includendo nella segnalazione "
"i file /var/log/dist-upgrade/main.log e /var/log/dist-upgrade/apt.log. "
"L'avanzamento si è interrotto.\n"
"Il file sources.list originale è stato salvato come /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:495
msgid "Ctrl-c pressed"
msgstr "Premuto Ctrl-C"

#: ../DistUpgrade/DistUpgradeViewGtk.py:483
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Questo terminerà l'operazione e potrebbe lasciare il sistema in uno stato "
"d'errore. Confermare l'operazione?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:631
#: ../DistUpgrade/DistUpgradeViewGtk3.py:629
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Chiudere tutte le applicazioni e i documenti aperti per prevenire la perdita "
"di dati."

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:643
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Non più supportati da Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Da retrocedere (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "Remove (%s)"
msgstr "Da rimuovere (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "No longer needed (%s)"
msgstr "Non più necessari (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "Install (%s)"
msgstr "Da installare (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Upgrade (%s)"
msgstr "Da aggiornare (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#: ../UpdateManager/DistUpgradeFetcherKDE.py:157
msgid "Media Change"
msgstr "Cambio del supporto"

#: ../DistUpgrade/DistUpgradeViewKDE.py:335
msgid "Show Difference >>>"
msgstr "Mostra differenze >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:338
msgid "<<< Hide Difference"
msgstr "<<< Nascondi differenze"

#: ../DistUpgrade/DistUpgradeViewKDE.py:554
msgid "Error"
msgstr "Errore"

#: ../DistUpgrade/DistUpgradeViewKDE.py:568
msgid "&Cancel"
msgstr "&Annulla"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "&Close"
msgstr "&Chiudi"

#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "Show Terminal >>>"
msgstr "Mostra terminale >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:621
msgid "<<< Hide Terminal"
msgstr "<<< Nascondi terminale"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Information"
msgstr "Informazioni"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
#: ../DistUpgrade/DistUpgradeViewKDE.py:796
#: ../DistUpgrade/DistUpgradeViewKDE.py:799 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Dettagli"

#: ../DistUpgrade/DistUpgradeViewKDE.py:777
#, python-format
msgid "No longer supported %s"
msgstr "Non più supportato %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "Remove %s"
msgstr "Rimuovere %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgradeViewText.py:182
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Rimuovere %s (installato automaticamente)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Install %s"
msgstr "Installare %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Upgrade %s"
msgstr "Avanzare %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
#: ../DistUpgrade/DistUpgradeViewText.py:230
msgid "Restart required"
msgstr "Riavvio richiesto"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Riavviare il sistema per completare l'avanzamento</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:812 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "_Riavvia ora"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:830
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Annullare l'avanzamento in corso?</big></b>\n"
"\n"
"Se viene annullato l'avanzamento, il sistema può rimanere in uno stato "
"instabile. È fortemente consigliato riprendere l'avanzamento."

#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid "Cancel Upgrade?"
msgstr "Annullare l'avanzamento di versione?"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li giorno"
msgstr[1] "%li giorni"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li ora"
msgstr[1] "%li ore"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuto"
msgstr[1] "%li minuti"

#: ../DistUpgrade/DistUpgradeView.py:66
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li secondo"
msgstr[1] "%li secondi"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s e %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:100
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Lo scaricamento richiede circa %s con una connessione DSL da 1Mbit e circa "
"%s con un modem 56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:155
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Questo scaricamento richiederà circa %s con la propria connessione. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Preparazione all'avanzamento di versione"

#: ../DistUpgrade/DistUpgradeView.py:260
msgid "Getting new software channels"
msgstr "Recupero nuovi canali software"

#: ../DistUpgrade/DistUpgradeView.py:261 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Recupero nuovi pacchetti"

#: ../DistUpgrade/DistUpgradeView.py:262 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Installazione degli aggiornamenti"

#: ../DistUpgrade/DistUpgradeView.py:263 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Pulizia"

#: ../DistUpgrade/DistUpgradeView.py:348
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d pacchetto installato non è più supportato da Canonical. È "
"possibile continuare a ottenere supporto dalla comunità."
msgstr[1] ""
"%(amount)d pacchetti installati non sono più supportati da Canonical. È "
"possibile continuare a ottenere supporto dalla comunità."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pacchetto sta per essere rimosso."
msgstr[1] "%d pacchetti stanno per essere rimossi."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nuovo pacchetto sta per essere installato."
msgstr[1] "%d nuovi pacchetti stanno per essere installati."

#: ../DistUpgrade/DistUpgradeView.py:368
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pacchetto sta per essere aggiornato."
msgstr[1] "%d pacchetti stanno per essere aggiornati."

#: ../DistUpgrade/DistUpgradeView.py:373
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"È necessario scaricare un totale di %s. "

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"L'installazione degli avanzamenti di versione può richiedere diverse ore e "
"non è possibile annullarlo in un momento successivo."

#: ../DistUpgrade/DistUpgradeView.py:382
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Il recupero dei file e l'installazione degli avanzamenti di versione possono "
"richiedere diverse ore e non è possibile annullarli in un momento successivo."

#: ../DistUpgrade/DistUpgradeView.py:387
msgid "Removing the packages can take several hours. "
msgstr "La rimozione dei pacchetti può richiedere diverse ore. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:392 ../UpdateManager/UpdateManager.py:676
msgid "The software on this computer is up to date."
msgstr "Il software è aggiornato."

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Non ci sono avanzamenti di versione disponibili per questo sistema. "
"L'avanzamento sarà annullato."

#: ../DistUpgrade/DistUpgradeView.py:406
msgid "Reboot required"
msgstr "Riavvio richiesto"

#: ../DistUpgrade/DistUpgradeView.py:407
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"L'avanzamento di versione è stato completato ed è richiesto un riavvio. "
"Riavviare ora?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:72
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autenticazione di «%(file)s» con «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#, python-format
msgid "extracting '%s'"
msgstr "estrazione di «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:151
#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:151
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid "Could not run the upgrade tool"
msgstr "Impossibile eseguire lo strumento di avanzamento versione"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Probabilmente c'è un bug nello strumento di avanzamento. Segnalare questo "
"problema tramite il comando «ubuntu-bug update-manager» da terminale."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Firma dello strumento di avanzamento versione"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Strumento di avanzamento versione"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Recupero non riuscito"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Il recupero dei file per l'avanzamento di versione non è riuscito. Potrebbe "
"dipendere da un problema di rete. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Autenticazione non riuscita"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Non è riuscita l'autenticazione dell'avanzamento di versione. Potrebbe "
"dipendere da un problema con la connessione di rete o con il server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Estrazione non riuscita"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Non è riuscita l'estrazione dell'aggiornamento. Potrebbe dipendere da un "
"problema con la connesione di rete o con il server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Verifica non riuscita"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Non è riuscita la verifica dell'avanzamento di versione. Potrebbe dipendere "
"da un problema con la connessione di rete o con il server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:300
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Impossibile eseguire l'avanzamento"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Questo di solito succede se /tmp è montata in modalità noexec. Montarla "
"senza modalità noexec ed eseguire nuovamente l'avanzamento."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Il messaggio di errore è «%s»."

#: ../DistUpgrade/DistUpgradeViewText.py:93
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Se non è già stato fatto, segnalare questo bug includendo nella segnalazione "
"i file /var/log/dist-upgrade/main.log e /var/log/dist-upgrade/apt.log. "
"L'avanzamento si è interrotto.\n"
"Il file sources.list originale è stato salvato come /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:117
msgid "Aborting"
msgstr "Interruzione"

#: ../DistUpgrade/DistUpgradeViewText.py:122
msgid "Demoted:\n"
msgstr "Retrocesso:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:129
msgid "To continue please press [ENTER]"
msgstr "Per continuare premere [INVIO]"

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [yN] "
msgstr "Continua [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Details [d]"
msgstr "Dettagli [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:165
#: ../DistUpgrade/DistUpgradeViewText.py:213
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:173
#, python-format
msgid "No longer supported: %s\n"
msgstr "Non più supportato: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, python-format
msgid "Remove: %s\n"
msgstr "Rimuove: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:188
#, python-format
msgid "Install: %s\n"
msgstr "Installa: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:193
#, python-format
msgid "Upgrade: %s\n"
msgstr "Avanza: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "Continue [Yn] "
msgstr "Continua [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:231
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Per completare l'avanzamento è necessario riavviare il computer.\n"
"Selezionando \"s\" il sistema sarà riavviato."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "A_nnula avanzamento"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Riprendi avanzamento"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Annullare l'avanzamento di versione in corso?</big></b>\n"
"\n"
"Annullando l'avanzamento, il sistema potrebbe trovarsi in uno stato "
"inutilizzabile. È fortemente consigliato riprendere l'avanzamento di "
"versione."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Avvia avanzamento"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Sostituisci"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Differenze tra i file"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Segnala bug"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continua"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Avviare l'avanzamento di versione?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Riavviare il sistema per completare l'avanzamento</big></b>\n"
"\n"
"Salvare il proprio lavoro prima di proseguire."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Avanzamento distribuzione"

#: ../DistUpgrade/DistUpgrade.ui.h:19
#, fuzzy
msgid "<b><big>Upgrading Ubuntu to version 12.10</big></b>"
msgstr "<b><big>Avanzamento di Ubuntu alla versione 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Impostazione nuovi canali software"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Riavvio del sistema"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Terminale"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:64
msgid "Please wait, this can take some time."
msgstr "Attendere, potrebbe richiedere del tempo."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:66
msgid "Update is complete"
msgstr "Aggiornamento completato"

#: ../UpdateManager/DistUpgradeFetcher.py:114
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Could not find the release notes"
msgstr "Impossibile trovare le note di rilascio"

#: ../UpdateManager/DistUpgradeFetcher.py:115
#: ../UpdateManager/DistUpgradeFetcherKDE.py:110
msgid "The server may be overloaded. "
msgstr "Il server potrebbe essere sovraccarico. "

#: ../UpdateManager/DistUpgradeFetcher.py:125
#: ../UpdateManager/DistUpgradeFetcherKDE.py:114
msgid "Could not download the release notes"
msgstr "Impossibile scaricare le note di rilascio"

#: ../UpdateManager/DistUpgradeFetcher.py:126
#: ../UpdateManager/DistUpgradeFetcherKDE.py:115
msgid "Please check your internet connection."
msgstr "Controllare la propria connessione a internet."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:68
#: ../UpdateManager/DistUpgradeFetcherKDE.py:91
msgid "Upgrade"
msgstr "Avanza"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:95
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Note di rilascio"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:134
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Downloading additional package files..."
msgstr "Scaricamento pacchetti aggiuntivi..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "File %s di %s a %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
#, python-format
msgid "File %s of %s"
msgstr "File %s di %s"

#: ../UpdateManager/ChangelogViewer.py:75
msgid "Open Link in Browser"
msgstr "Apri collegamento nel browser"

#: ../UpdateManager/ChangelogViewer.py:78
msgid "Copy Link to Clipboard"
msgstr "Copia collegamento negli appunti"

#: ../UpdateManager/GtkProgress.py:162
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Scaricamento del file %(current)li di %(total)li a %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:167
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Scaricamento del file %(current)li di %(total)li"

#: ../UpdateManager/UpdateManager.py:106 ../do-release-upgrade:100
msgid "Your Ubuntu release is not supported anymore."
msgstr "Il rilascio di Ubuntu in uso non è più supportato."

#: ../UpdateManager/UpdateManager.py:107
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""
"Non saranno disponibili futuri aggiornamenti critici o di sicurezza. "
"Effettuare l'aggiornamento a una versione più recente di Ubuntu."

#: ../UpdateManager/UpdateManager.py:115
msgid "Upgrade information"
msgstr "Informazioni avanzamento"

#: ../UpdateManager/UpdateManager.py:233
#: ../UpdateManagerText/UpdateManagerText.py:35
msgid "Install"
msgstr "Installa"

#: ../UpdateManager/UpdateManager.py:235
msgid "Name"
msgstr "Nome"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:395
#, python-format
msgid "Version %s: \n"
msgstr "Versione %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr "Nessuna connessione di rete, impossibile scaricare il changelog."

#: ../UpdateManager/UpdateManager.py:463
msgid "Downloading list of changes..."
msgstr "Scaricamento dell'elenco dei cambiamenti..."

#: ../UpdateManager/UpdateManager.py:507
msgid "_Deselect All"
msgstr "_Deseleziona tutto"

#: ../UpdateManager/UpdateManager.py:513
msgid "Select _All"
msgstr "_Seleziona tutto"

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr "Verranno scaricati %s."

#: ../UpdateManager/UpdateManager.py:584
#, fuzzy
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] "L'aggiornamento è stato scaricato, ma non installato."
msgstr[1] "Gli aggiornamenti sono stati scaricati, ma non installati."

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr "Non ci sono aggiornamenti da installare."

#: ../UpdateManager/UpdateManager.py:598
msgid "Unknown download size."
msgstr "Dimensione scaricamento sconosciuta."

#: ../UpdateManager/UpdateManager.py:624
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Il momento dell'ultima verifica è sconosciuto. Fare clic sul pulsante "
"«Verifica» per aggiornare le informaizoni."

#: ../UpdateManager/UpdateManager.py:630
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Le informazioni sui pacchetti sono state aggiornate %(days_ago)s giorni fa.\n"
"Premere il pulsante «Verifica» per controllare la presenza di nuovi "
"aggiornamenti software."

#: ../UpdateManager/UpdateManager.py:635
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural "The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Le informazioni sui pacchetti sono state aggiornate l'ultima volta un giorno "
"fa."
msgstr[1] ""
"Le informazioni sui pacchetti sono state aggiornate l'ultima volta "
"%(days_ago)s giorni fa."

#: ../UpdateManager/UpdateManager.py:639
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Le informazioni sui pacchetti sono state aggiornate l'ultima volta un'ora fa."
msgstr[1] ""
"Le informazioni sui pacchetti sono state aggiornate l'ultima volta "
"%(hours_ago)s ore fa."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:644 ../UpdateManager/UpdateManager.py:646
#: ../UpdateManager/UpdateManager.py:648
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr ""
"Le informazioni sui pacchetti sono state aggiornate circa %s minuti fa."

#: ../UpdateManager/UpdateManager.py:650
msgid "The package information was just updated."
msgstr "Le informazioni sui pacchetti sono appena state aggiornate."

#: ../UpdateManager/UpdateManager.py:689
msgid "Software updates may be available for your computer."
msgstr "Potrebbero essere disponibili aggiornamenti software."

#: ../UpdateManager/UpdateManager.py:697
#, python-format
msgid ""
"Updated software has been issued since %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:700
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:758
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"L'avanzamento necessita di %s di spazio libero sul disco «%s»: liberare "
"almeno altri %s di spazio sul disco «%s». Svuotare il cestino e rimuovere i "
"pacchetti temporanei di precedenti installazioni con il comando «sudo apt-"
"get clean»."

#: ../UpdateManager/UpdateManager.py:783
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Per concludere l'installazione degli aggiornamenti è necessario riavviare il "
"computer. Salvare il proprio lavoro prima di continuare."

#: ../UpdateManager/UpdateManager.py:847
msgid "Reading package information"
msgstr "Lettura informazioni sui pacchetti"

#: ../UpdateManager/UpdateManager.py:862
msgid "Connecting..."
msgstr "Connessione..."

#: ../UpdateManager/UpdateManager.py:879
msgid "You may not be able to check for updates or download new updates."
msgstr "Potrebbe non essere possibile controllare e scaricare aggiornamenti."

#: ../UpdateManager/UpdateManager.py:1002
msgid "Could not initialize the package information"
msgstr "Impossibile inizializzare le informazioni del pacchetto"

#: ../UpdateManager/UpdateManager.py:1003
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Si è verificato un problema irrisolvibile durante l'inizializzazione delle "
"informazioni del pacchetto.\n"
"\n"
"Segnalare questo bug per il pacchetto «update-manager» e includere il "
"seguente messaggio d'errore:\n"

#: ../UpdateManager/UpdateManager.py:1032
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Si è verificato un problema irrisolvibile durante il calcolo "
"dell'avanzamento.\n"
"\n"
"Segnalare questo bug per il pacchetto «update-manager» e includere il "
"seguente messaggio di errore:"

#: ../UpdateManager/UpdateManager.py:1056
msgid " (New install)"
msgstr " (Nuova installazione)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1063
#, python-format
msgid "(Size: %s)"
msgstr "(Dimensione: %s)"

#: ../UpdateManager/UpdateManager.py:1067
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Dalla versione %(old_version)s alla %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1071
#, python-format
msgid "Version %s"
msgstr "Versione %s"

#: ../UpdateManager/UpdateManager.py:1104 ../do-release-upgrade:112
msgid "Release upgrade not possible right now"
msgstr "Avanzamento di versione impossibile in questo momento"

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:113
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"L'avanzamento di versione non può essere eseguito, riprovare in un secondo "
"momento. Il messaggio del server è stato: «%s»"

#: ../UpdateManager/UpdateManager.py:1107 ../check-new-release-gtk:126
msgid "Downloading the release upgrade tool"
msgstr "Scaricamento dello strumento di avanzamento versione"

#: ../UpdateManager/UpdateManager.py:1114
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>È disponibile il nuovo rilascio «%s» di Ubuntu</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1153
msgid "Software index is broken"
msgstr "L'indice del software è rovinato"

#: ../UpdateManager/UpdateManager.py:1154
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Impossibile installare o rimuovere alcun software. Utilizzare il gestore dei "
"pacchetti «Synaptic» o eseguire «sudo apt-get install -f» in un terminale "
"per correggere innanzitutto questo problema."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Verifica aggiornamenti"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Installa aggiornamenti disponibili"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Cancel"
msgstr "Annulla"

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Changelog"
msgstr "Changelog"

#: ../UpdateManagerText/UpdateManagerText.py:40
msgid "Updates"
msgstr "Aggiornamenti"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid "Building Updates List"
msgstr "Creazione elenco aggiornamenti"

#: ../UpdateManagerText/UpdateManagerText.py:56
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Impossibile calcolare un avanzamento normale, eseguire: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Questo può essere causato da:\n"
" * un precedente avanzamento di versione che non è stato completato;\n"
" * problemi con alcuni software installati;\n"
" * pacchetti software non ufficiali non forniti da Ubuntu;\n"
" * normali cambiamenti di una versione di pre-rilascio di Ubuntu."

#: ../UpdateManagerText/UpdateManagerText.py:125
msgid "Downloading changelog"
msgstr "Scaricamento changelog"

#: ../UpdateManager/Core/MyCache.py:147
#, python-format
msgid "Other updates (%s)"
msgstr "Altri aggiornamenti (%s)"

#: ../UpdateManager/Core/MyCache.py:325
msgid "This update does not come from a source that supports changelogs."
msgstr ""
"Questo aggiornamento non proviene da una fonte che supporta i changelog."

#: ../UpdateManager/Core/MyCache.py:331 ../UpdateManager/Core/MyCache.py:359
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Scaricamento dell'elenco delle modifiche non riuscito. \n"
"Verificare la connessione a Internet."

#: ../UpdateManager/Core/MyCache.py:338
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Modifiche per le versioni:\n"
"Versione installata: %s\n"
"Versione disponibile: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:348
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Il changelog non contiene alcuna variazione rilevante.\n"
"\n"
"Consultare http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"finché i cambiamenti non sono disponibili o riprovare più tardi."

#: ../UpdateManager/Core/MyCache.py:353
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"L'elenco dei cambiamenti non è ancora disponibile.\n"
"\n"
"Utilizzare http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"finché le modifiche non saranno disponibili oppure riprovare più tardi."

#: ../UpdateManager/Core/UpdateList.py:51
msgid "Failed to detect distribution"
msgstr "Identificazione della distribuzione non riuscita"

#: ../UpdateManager/Core/UpdateList.py:52
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr ""
"Si è verificato un errore «%s» durante il controllo del sistema in uso."

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Important security updates"
msgstr "Aggiornamenti di sicurezza importanti"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Recommended updates"
msgstr "Aggiornamenti raccomandati"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Proposed updates"
msgstr "Aggiornamenti proposti"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Backports"
msgstr "Backport"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Distribution updates"
msgstr "Aggiornamenti della distribuzione"

#: ../UpdateManager/Core/UpdateList.py:72
msgid "Other updates"
msgstr "Altri aggiornamenti"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
#, fuzzy
msgid "<big><b>Starting Software Updater</b></big>"
msgstr "<big><b>Avvio del gestore di aggiornamenti</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Gli aggiornamenti software correggono errori, eliminano vulnerabilità di "
"sicurezza e forniscono nuove funzionalità."

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "Esegui avanzamento _parziale"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr ""
"<big><b>Non tutti gli aggiornamenti possono essere installati</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Eseguire un avanzamento di versione parziale per installare più "
"aggiornamenti possibile. \n"
"\n"
"Le cause possono essere:\n"
" * Un avanzamento precedente non completato\n"
" * Problemi con alcuni programmi installati\n"
" * Pacchetti software non ufficiali non forniti da Ubuntu\n"
" * Normali cambiamenti di una versione pre-rilascio di Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "_Verifica"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>È necessario controllare gli aggiornamenti manualmente</big></b>\n"
"\n"
"Il controllo automatico della disponibilità di aggiornamenti non è attivo. È "
"possibile configurare questo comportamento in <i>Sorgenti software</i> nella "
"scheda <i>Aggiornamenti</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "_Non mostrare più queste informazioni"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "_Continua"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Alimentazione da batteria</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Il sistema è alimentato dalla batteria. Continuare comunque?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Esegui avanzamento"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Mostra avanzamento dei singoli file"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
#, fuzzy
msgid "Software Updater"
msgstr "Aggiornamenti software"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
#, fuzzy
msgid "Starting Software Updater"
msgstr "Aggiornamenti software"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "_Esegui avanzamento"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "aggiornamenti"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Changes"
msgstr "Cambiamenti"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "Description"
msgstr "Descrizione"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
#, fuzzy
msgid "Details of updates"
msgstr "Descrizione dell'aggiornamento"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Connessione eseguita in roaming: è possibile che il costo sia basato sui "
"dati scaricati per l'aggiornamento."

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Per sicurezza collegare il computer alla rete elettrica prima di eseguire "
"aggiornamenti."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Settings..."
msgstr "_Impostazioni..."

#: ../data/gtkbuilder/UpdateManager.ui.h:35
#, fuzzy
msgid "_Install Now"
msgstr "Installa"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>È disponibile una nuova versione di Ubuntu. Effettuare l'avanzamento?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Non avanzare"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Ricorda in seguito"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Esegui avanzamento"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Avanzamento alla nuova versione di Ubuntu rifiutato"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
#, fuzzy
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"È possibile effettuare l'avanzamento in un secondo momento aprendo il "
"gestore aggiornamenti e facendo clic su «Avanzamento»."

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Aggiornamenti software"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Mostra e installa gli aggiornamenti disponibili"

#: ../update-manager:66 ../update-manager-text:55 ../do-release-upgrade:51
msgid "Show version and exit"
msgstr "Mostra la versione ed esce"

#: ../update-manager:69
msgid "Directory that contains the data files"
msgstr "Directory contenente i file di dati"

#: ../update-manager:72
msgid "Check if a new Ubuntu release is available"
msgstr "Controlla la disponibilità di un nuovo rilascio di Ubuntu"

#: ../update-manager:75 ../do-release-upgrade:54 ../check-new-release-gtk:186
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Verifica se è possibile avanzare all'ultima versione di sviluppo"

#: ../update-manager:79
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Usa l'ultima versione proposta del sistema di avanzamento di versione"

#: ../update-manager:86
msgid "Do not focus on map when starting"
msgstr "Avvia senza dare il focus alla finestra del gestore"

#: ../update-manager:89
msgid "Try to run a dist-upgrade"
msgstr "Prova a eseguire un dist-upgrade"

#: ../update-manager:92
msgid "Do not check for updates when starting"
msgstr "Non controlla la presenza di aggiornamenti all'avvio"

#: ../update-manager:96 ../do-release-upgrade:70
msgid "Test upgrade with a sandbox aufs overlay"
msgstr ""
"Prova di avanzamento in ambiente sandbox con sovrapposizione di file system "
"aufs"

#: ../update-manager:116
msgid "Running partial upgrade"
msgstr "Esecuzione avanzamento di versione parziale"

#: ../update-manager-text:59
msgid "Show description of the package instead of the changelog"
msgstr "Mostra la descrizione del pacchetto al posto del changelog"

#: ../do-release-upgrade:58 ../check-new-release-gtk:190
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Tentativo di avanzamento all'ultimo rilascio usando lo strumento di "
"avanzamento da $distro-proposed"

#: ../do-release-upgrade:62
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Eseguire in modalità speciale di avanzamento.\n"
"Attualmente sono supportati «desktop» per un avanzamento normale di sistemi "
"desktop e «server» per sistemi server."

#: ../do-release-upgrade:68
msgid "Run the specified frontend"
msgstr "Esegue il frontend specificato"

#: ../do-release-upgrade:73
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Controlla se è dispoibile un nuovo rilascio della distribuzione e riporta il "
"risultato tramite un codice d'uscita"

#: ../do-release-upgrade:87
msgid "Checking for a new Ubuntu release"
msgstr "Verifica un nuovo rilascio di ubuntu"

#: ../do-release-upgrade:101
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Per informazioni sull'avanzamento consultare:\n"
"$(url)s\n"

#: ../do-release-upgrade:107
msgid "No new release found"
msgstr "Nessun nuovo rilascio trovato"

#: ../do-release-upgrade:119
#, c-format
msgid "New release '%s' available."
msgstr "Nuovo rilascio «%s» disponibile."

#: ../do-release-upgrade:120
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Lanciare «do-release-upgrade» per eseguire l'avanzamento."

#: ../check-new-release-gtk:101
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Disponibile avanzamento a Ubuntu &(version)s"

#: ../check-new-release-gtk:143
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "È stato rifiutato l'avanzamento alla versione %s di Ubuntu"

#: ../check-new-release-gtk:196
msgid "Add debug output"
msgstr "Abilitare le informazioni di debug"

#: ../ubuntu-support-status:91
msgid "Show unsupported packages on this machine"
msgstr "Mostra pacchetti non supportati"

#: ../ubuntu-support-status:94
msgid "Show supported packages on this machine"
msgstr "Mostra pacchetti supportati"

#: ../ubuntu-support-status:97
msgid "Show all packages with their status"
msgstr "Mostra lo stato di tutti i pacchetti"

#: ../ubuntu-support-status:100
msgid "Show all packages in a list"
msgstr "Mostra tutti i pacchetti in un elenco"

#: ../ubuntu-support-status:142
#, c-format
msgid "Support status summary of '%s':"
msgstr "Riepilogo supporto per «%s»:"

#: ../ubuntu-support-status:145
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr "Ci sono %(num)s pacchetti (%percent).1f%%) supportati fino a %(time)s"

#: ../ubuntu-support-status:151
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr "Ci sono %(num)s pacchetti (%(percent).1f%%) non più scaricabili"

#: ../ubuntu-support-status:154
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr "Ci sono %(num)s pacchetti (%(percent).1f%%) non supportati"

#: ../ubuntu-support-status:162
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""
"Eseguire con --show-unsupported, --show-supported o --show-all per ulteriori "
"dettagli"

#: ../ubuntu-support-status:166
msgid "No longer downloadable:"
msgstr "Non più scaricabili:"

#: ../ubuntu-support-status:169
msgid "Unsupported: "
msgstr "Non supportati: "

#: ../ubuntu-support-status:174
#, c-format
msgid "Supported until %s:"
msgstr "Supportati fino a %s:"

#: ../ubuntu-support-status:183
msgid "Unsupported"
msgstr "Non supportato"

#. Why do we use %s here instead of $strings or {} format placeholders?
#. It's because we don't want to break existing translations.
#: ../janitor/plugincore/exceptions.py:42
#, python-format
msgid "Unimplemented method: %s"
msgstr "Metodo non implementato: %s"

#: ../janitor/plugincore/core/file_cruft.py:41
msgid "A file on disk"
msgstr "Un file su disco"

#: ../janitor/plugincore/core/missing_package_cruft.py:39
msgid "Install missing package."
msgstr "Installare il pacchetto mancante."

#. 2012-06-08 BAW: i18n string; don't use {} or PEP 292.
#: ../janitor/plugincore/core/missing_package_cruft.py:49
#, python-format
msgid "Package %s should be installed."
msgstr "È necessario installare il pacchetto %s."

#: ../janitor/plugincore/core/package_cruft.py:49
msgid ".deb package"
msgstr "Pacchetto .deb"

#: ../janitor/plugincore/plugins/langpack_manual_plugin.py:46
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "È necessario selezionare il pacchetto %s come installato manualmente."

#: ../janitor/plugincore/plugins/kdelibs4to5_plugin.py:49
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Se è installato kdelibs4-dev, durante l'avanzamento sarà necessario "
"installare kdelibs5-dev. Per maggiori informazioni, consultare il bug n° "
"279621 su bugs.launchpad.net."

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:44
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i voci non aggiornate nel file di stato"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:47
msgid "Obsolete entries in dpkg status"
msgstr "Voci non aggiornate nello stato di dpkg"

#. pragma: no cover
#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:50
msgid "Obsolete dpkg status entries"
msgstr "Voci dello stato di dpkg non aggiornate"

#: ../janitor/plugincore/plugins/remove_lilo_plugin.py:42
msgid "Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"È necessario rimuovere lilo per la presenza di grub. Per ulteriori dettagli "
"consultare il bug n° 314004."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore so a bug reporting process is being started."
#~ msgstr ""
#~ "Dopo l'aggiornamento delle informazioni sui pacchetti, il pacchetto "
#~ "essenziale «%s» non può più essere trovato. Sarà ora possibile segnalare "
#~ "questo problema."

#~ msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
#~ msgstr "<b><big>Avanzamento di Ubuntu alla versione 12.04</big></b>"

#~ msgid "%(count)s update has been selected."
#~ msgid_plural "%(count)s updates have been selected."
#~ msgstr[0] "È stato selezionato un aggiornamento."
#~ msgstr[1] "Sono stati selezionati %(count)s aggiornamenti."

#~ msgid "%(count_str)s %(download_str)s"
#~ msgstr "%(count_str)s %(download_str)s"

#~ msgid "Welcome to Ubuntu"
#~ msgstr "Benvenuti in Ubuntu"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released."
#~ msgstr ""
#~ "Questi aggiornamenti sono stati resi disponibili dopo il rilascio di "
#~ "questa versione di Ubuntu."

#~ msgid "Software updates are available for this computer."
#~ msgstr "Sono disponibili aggiornamenti software."

#~ msgid "Update Manager"
#~ msgstr "Gestore aggiornamenti"

#~ msgid "Starting Update Manager"
#~ msgstr "Avvio del Gestore aggiornamenti"

#~ msgid "You are connected via a wireless modem."
#~ msgstr "Connessione eseguita attraverso un modem senza fili."

#~ msgid "_Install Updates"
#~ msgstr "I_nstalla aggiornamenti"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Verifica un nuovo rilascio di ubuntu"

#~ msgid "Your system is up-to-date"
#~ msgstr "Il sistema è aggiornato"

#~ msgid "Software updates are available for this computer"
#~ msgstr "Sono disponibili aggiornamenti software"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Questo avanzamento viene eseguito in modalità \"sandbox\" (di prova). "
#~ "Tutte le modifiche vengono scritte in «%s» e al prossimo riavvio andranno "
#~ "perse.\n"
#~ "\n"
#~ "Nessuna modifica salvata in una cartella di sistema, da ora al prossimo "
#~ "riavvio, è permanente."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Per non installarli ora, scegliere in un secondo momento Sistema → "
#~ "Amministrazione → Gestore aggiornamenti."

#~ msgid "There are no updates to install"
#~ msgstr "Non ci sono aggiornamenti da installare"

#~ msgid "The update has already been downloaded, but not installed"
#~ msgid_plural "The updates have already been downloaded, but not installed"
#~ msgstr[0] "L'aggiornamento è stato scaricato, ma non installato"
#~ msgstr[1] "Gli aggiornamenti sono stati scaricati, ma non installati"

#~ msgid ""
#~ "You will not get any further security fixes or critical updates. Please "
#~ "upgrade to a later version of Ubuntu Linux."
#~ msgstr ""
#~ "Non saranno disponibili futuri aggiornamenti critici o di sicurezza. Si "
#~ "raccomanda di effettuare l'aggiornamento a una versione più recente di "
#~ "Ubuntu Linux."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from "
#~ "Applications later."
#~ msgstr ""
#~ "Per non installarli ora, scegliere in un secondo momento Applicazioni → "
#~ "Gestore aggiornamenti."

#~ msgid "%.0f kB"
#~ msgstr "%.0f kB"

#~ msgid "0 kB"
#~ msgstr "0 kB"

#~ msgid "1 kB"
#~ msgstr "1 kB"

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The "
#~ "upgrade will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug using the command 'ubuntu-bug update-manager' "
#~ "in a terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Non è stato possibile ottenere i prerequisiti per l'avanzamento. Verrà "
#~ "ora ripristinato lo stato originario del sistema.\n"
#~ "\n"
#~ "Segnalare questo problema tramite il comando «ubuntu-bug update-manager» "
#~ "da terminale e includere i file contenuti in /var/log/dist-upgrade/ nella "
#~ "segnalazione."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal."
#~ msgstr ""
#~ "L'aggiornamento delle informazioni sui repository ha creato un file non "
#~ "valido. Segnalare questo problema tramite il comando «ubuntu-bug update-"
#~ "manager» da terminale."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug using the command "
#~ "'ubuntu-bug update-manager' in a terminal and include the files in /var/"
#~ "log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Dopo l'aggiornamento delle informazioni sui pacchetti, il pacchetto "
#~ "essenziale «%s» non può più essere trovato.\n"
#~ "Questo indica un errore grave, segnalare questo problema tramite il "
#~ "comando «ubuntu-bug update-manager» da terminale e includere i file "
#~ "contenuti in /var/log/dist-upgrade/ nella segnalazione."

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Segnalare questo problema tramite il comando «ubuntu-bug update-manager» "
#~ "da terminale e includere i file contenuti in /var/log/dist-upgrade/ nella "
#~ "segnalazione.\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this using the "
#~ "command 'ubuntu-bug update-manager' in a terminal and include the files "
#~ "in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "La preparazione del sistema per l'avanzamento non è riuscita. Segnalare "
#~ "questo problema tramite il comando «ubuntu-bug update-manager» da "
#~ "terminale e includere i file contenuti in /var/log/dist-upgrade/ nella "
#~ "segnalazione."

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
#~ msgstr ""
#~ "La scheda video potrebbe non essere pienamente supportata in Ubuntu 11.10."

#~ msgid ""
#~ "The support in Ubuntu 11.04 for your intel graphics hardware is limited "
#~ "and you may encounter problems after the upgrade. Do you want to continue "
#~ "with the upgrade?"
#~ msgstr ""
#~ "Il supporto per le schede video Intel in Ubuntu 11.10 è limitato e "
#~ "potrebbero esserci dei problemi dopo l'avanzamento. Continuare?"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from Applications later."
#~ msgstr ""
#~ "Questi aggiornamenti sono stati resi disponibili dopo il rilascio di "
#~ "questa versione di Ubuntu. Per non installarli ora, scegliere in un "
#~ "secondo momento Applicazioni → Gestore aggiornamenti."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from the Administration Menu later."
#~ msgstr ""
#~ "Questi aggiornamenti sono stati resi disponibili dopo il rilascio di "
#~ "questa versione di Ubuntu. Per non installarli ora, selezionare in un "
#~ "secondo momento Sistema → Amministrazione → Gestore aggiornamenti."

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Il recupero dei file e l'installazione degli avanzamenti di versione "
#~ "possono richiedere diverse ore e non è possibile annullarli in un momento "
#~ "successivo."
